cinema/models.py

```diff
@@ -110,18 +131,18 @@ def clean(self):
         )
 
     def save(
-        self,
-        force_insert=False,
-        force_update=False,
-        using=None,
-        update_fields=None,
-    ):
+            self,
+            force_insert: bool = False,
+            force_update: bool = False,
+            using: Any = None,
+            update_fields: Any = None,
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
using and update_fields types is not null